spacy/pipeline/tok2vec.py

```diff
@@ -286,8 +283,20 @@ def verify_inputs(self, inputs) -> bool:
 def forward(model: Tok2VecListener, inputs, is_train: bool):
     """Supply the outputs from the upstream Tok2Vec component."""
     if is_train:
-        model.verify_inputs(inputs)
-        return model._outputs, model._backprop
+        # This might occur during training when the tok2vec layer is frozen / hasn't been updated.
+        # In that case, it should be set to "annotating" so we can retrieve the embeddings from the doc.
+        if model._batch_id is None:
+            outputs = []
+            for doc in inputs:
+                if doc.tensor.size == 0:
+                    # TODO move to errors.py
+                    raise ValueError("If the embedding layer is not updated during training, include it in 'annotating components'")
```

Comment:
`embedding layer` meaning the `tok2vec` ?

Reply:
Yea. We can be more specific and say `tok2vec/transformer layer` instead (I want to reuse the error msg for transformers, too).